src/key_io.cpp

```diff
@@ -95,7 +95,6 @@ CTxDestination DecodeDestination(const std::string& str, const CChainParams& par
     bool is_bech32 = bech32_encoding != bech32::Encoding::INVALID;
 
     auto check_base58 = [&]() { return DecodeBase58Check(str, data, MAX_BASE58_CHECK_CHARS); };
-    
```

Comment:
Nit for c974f892c7b6a61630c555f08ec201d911d5fd62 _Bech32 decoding logic_:  unrelated whitespace change and random looking `asdas` in the commit message.